docs/about/features_index/fed_eval.rst

```diff
@@ -42,468 +42,148 @@ With OpenFL version==1.7 aggregator start command is enhanced to have an optiona
     task_group (str): Selected task-group for assignement - defaults to 'learning'

```

Comment:
Fix spelling and the parameter options should be checked here: and the default for `task_group` is [no longer 'learning'](https://github.com/securefederatedai/openfl/blob/develop/openfl/interface/aggregator.py#L70-L73)